Cruzoft/nbminer-reporter
# Updated README file and added dashboard examples

# What does this pull request do?
It adds some examples on how to use the data on Influx and Grafana dashboards.

It also adds a small log entry to make debugging easier. 

# How should it be tested?

use the template files to create dashboards and check you can actually see data.

# Is related to an Issue?

* Closses #26 


# Any extra info?
